src/import/saga.ts

```diff
@@ -166,9 +166,11 @@ export function* importBackupPhraseSaga({ phrase, useEmptyWallet }: ImportBackup
     const skipVerification = yield select(skipVerificationSelector)
     if (skipVerification || numberAlreadyVerifiedCentrally) {
       yield put(setHasSeenVerificationNux(true))
+      // navigateHome will clear onboarding Stack
       navigateHome()
     } else {
-      navigateClearingStack(Screens.VerificationEducationScreen)
+      // DO NOT CLEAR NAVIGATION STACK HERE - breaks restore flow on initial app open in native-stack v6
+      navigate(Screens.VerificationEducationScreen)
```

Comment:
Does this comment apply to both iOS and Android?

Reply:
Yes - clearing the stack here was causing issues on both iOS and Android. Using the `UseBackToWelcomeScreen` was only causing issues on Android.